code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -609,9 +610,14 @@ func weightedAverage(prev, next *Terms, tx, pexp, expDiff common.Timestamp,
 
 // The adjustChallengePool moves more or moves some tokens back from or to
```

Comment:
Could we write comment follow https://blog.golang.org/godoc ? it is helpful for us to generate godoc in the future.

Reply:
Adding comments for pre-existing code is out of the scope. Plus this is an internal function so should not need comments for godoc, but only to help confused developers, I would expect only the endpoint methods linked in the `sc.go` files would need a godoc explanation.